proxy/delete.go

```diff
@@ -15,13 +15,13 @@ import (
 )
 
 // DeleteFunction delete a function from the FaaS server
-func DeleteFunction(gateway string, functionName string) error {
+func DeleteFunction(gateway string, functionName string, tlsInsecure bool) error {
 	gateway = strings.TrimRight(gateway, "/")
 	delReq := requests.DeleteFunctionRequest{FunctionName: functionName}
 	reqBytes, _ := json.Marshal(&delReq)
 	reader := bytes.NewReader(reqBytes)
 
-	c := http.Client{}
```

Comment:
I'm surprised to see this. I'm glad you spent more time looking into the code so we create the clients with the same options everywhere.